_data/doc_categories.yml

```diff
@@ -0,0 +1,12 @@
+# The order of the categories in this file (first-to-last) determines their
+# order in the UI (left-to-right)
+- title: Overview
+  href: docs
```

Comment:
This data file represents a mapping for the docs page category headers. These cannot be inferred from permalink front-matter nor filename, therefore they must be explicitly specified as a site relative link.